scripts/push-tutorials.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+setup_git() {
+  git config --global user.email "<>"
+  git config --global user.name "snork2d2"
```

Comment:
that's great :p

Reply:
https://github.com/snork2d2

Reply:
@vincentschen if you have some spare cycles and a photoshop license...